### ISSUE TITLE
About render speed

### ISSUE THREAD

**Issue by DrokBing** [graphdeco-inria/gaussian-splatting#278]
I want to max the render speed, but when i try -r 8, it's even slower than -r 2.

**Comment by Snosixtyboo** [comment]
Render or training speed?

**Comment by DrokBing** [comment]
> Render or training speed?

render
for example i use 
python render.py -m ./output/845a2c24-f --skip_train -r 8
is even slower than
python render.py -m ./output/845a2c24-f --skip_train -r 2

**Comment by Snosixtyboo** [comment]
Very interesting, currently makes no sense to me. Did you check that the resulting images have the desired resolution (1/8 and 1/2 respectively)? If they do I have no explanation right now.

**Comment by DrokBing** [comment]
yeah the resolution is right.
Btw, is there any parameter i can set to maximize render speed if i don't care about quality?

**Comment by Snosixtyboo** [comment]
No, unfortunately, there is none that i can think of. Since we rasterize, the only way to reduce the number of samples is to reduce the resolution.

**Comment by DrokBing** [comment]
Is there anyway to change the resolution? not use -r
